### DISCUSSION TITLE
Prototype Pollution in immer

### DISCUSSION THREAD

**Discussion by solankirohan** [jaredpalmer/razzle#1815] [Q&A] [answered]
Hello all, 

I updated razzle to the latest version 4.2.14 and when I run "npm audit" it shows 1 Critical error. See below:-

Critical: Prototype Pollution in immer
Package: immer
Patched in: >= 9.0.6
Dependency of: razzle[dev]
Path: razzle > react-dev-utils > immer

I wanted to know how to resolve this critical error even after updating razzle to the latest version?

Thank you!

**Comment by fivethreeo** [comment, marked as answer]
It is only part of dev so should not be a issue. But I cannot ugrade react-dev-utils in razzle 4. If you use yarn you can use resolutions to force immer version. Not sure if that is compatible with the react-dev-utils version.

**Reply by solankirohan** [reply]
thank you @fivethreeo for your response. I am not using yarn :(
